utils/vale/styles/Crossplane/crossplane-words.txt

```diff
@@ -63,4 +63,9 @@ XRC
 XRD
 XRD's
 XRDs
-XRs
\ No newline at end of file
+XRs
+P&T
+deployment-runtime
+pipeline-composition
+FunctionRefName
```

Comment:
```suggestion
```

This is a specific field inside a manifest, so it should be in code blocks and wouldn't be caught by vale (I think)